cli/build.gradle

```diff
@@ -268,6 +269,12 @@ test {
 	maxParallelForks = forks
 }
 
+unzipManifest {
+	from zipTree("${buildDir}/libs/blade.jar")
+	into "$buildDir/classes/java/main/"
```

Comment:
Can you refer to this using a property accessor?  Like `sourceSets.main.outputDir` or something?  that way if we ever change it in another location this location will get updated?